src/components/language-select/_language-select.scss

```diff
@@ -1,7 +1,5 @@
 .hmrc-language-select {
-  @include govuk-typography-responsive($size: 16);
-  @include govuk-link-common;
-  @include govuk-link-style-text;
+  @include govuk-font($size: 16);
```

Comment:
this class is used not on the language select links, but on the container, so link styles aren't needed, however we need govuk-font styles to replace them because the language select has spans for the current item - which have no govuk class. This does not change the current language names colour, if -I wonder- maybe the link styles were originally intended to make the current language look like a link 🤷‍♂️